### PR TITLE
Update github-vs-sourcegraph.md

### PR DESCRIPTION
**Pull request by lcolston08**
Added a link to the comparison doc & added changes based on 4.2 including audit logs and Smart Search.

## Test plan

Review copy

### CODE REVIEW

**Review by iskyOS — APPROVED** [review]
Looks good! Just made some very small suggestions. I think the audit log part could probably be omitted from this page too -- sent more info in Slack.

File: doc/getting-started/github-vs-sourcegraph.md [hunk 1]
```diff
@@ -566,7 +570,7 @@ GitHub does not offer comprehensive insights that account for the content of the
...
 
-Sourcegraph offers [in-product analytics](https://docs.sourcegraph.com/admin/analytics) to help Sourcegraph administrators understand user engagement across the various Sourcegraph features, identify power users, and convey value to engineering leaders. 
+Sourcegraph offers [in-product analytics](https://docs.sourcegraph.com/admin/analytics) to help Sourcegraph administrators understand user engagement across the various Sourcegraph features, identify power users, and convey value to engineering leaders. Easy-to-consume audit logs which include security events, Gitserver access events, monitoring, pings, and GraphQL requests is avialble for Sourcegraph self-hosted instances. Audit logs are coming soon for Sourcegraph Cloud.
```
Comment (lcolston08) [inline doc/getting-started/github-vs-sourcegraph.md:573]:
```suggestion
Sourcegraph offers [in-product analytics](https://docs.sourcegraph.com/admin/analytics) to help Sourcegraph administrators understand user engagement across the various Sourcegraph features, identify power users, and convey value to engineering leaders.

**Review by thisisvic — COMMENTED** [review]
typo in documennt

**Review by thisisvic — COMMENTED** [review]
Maybe split this into 2 sentences?

**Pull request merged**